Release 1.12 plan.md

```diff
@@ -0,0 +1,33 @@
+# Plan for Version 1.12
+
+## Timing
+
+The last feature release of chakracore occurred on 29th June 2018 (version 1.10) since then there have been numerous security update releases and 1 non-security bug fix release (1.11). There has however been significant feature work performed since that date all of which is unreleased.
+
+We would therefore like our first release to be done fairly quickly - however there are also some half complete pieces of work to finish before that as well as a variety of bugs introduced that should be fixed before that.
+
+We will aim to do this first release by June to allow time for confirming the exact status of the project, completing unfinished work, performing testing and fixing bugs.
```

Comment:
In a way I'd like to say End of April - but I don't think that's feasible, unless we could have one or two people full time. Anyone else's thoughts here appreciated.

Reply:
Agreed. I think June is a good goal for now.